src/main/java/uk/gov/hmcts/divorce/common/notification/SoleApplicationDisputedNotification.java

```diff
@@ -153,7 +137,13 @@ private Map<String, String> solicitorTemplateVars(CaseData caseData, Long id, Ap
 
         templateVars.put(ISSUE_DATE_PLUS_37_DAYS,
             caseData.getApplication().getIssueDate().plusDays(disputeDueDateOffsetDays).format(DATE_TIME_FORMATTER));
+        templateVars.put(ISSUE_DATE_PLUS_141_DAYS, ""); // Why is this here?? Should it be populated or removed?
```

Comment:
I think this needs to be checked out - if it's not put in there does the template bomb out if it's null instead of "" maybe?

Reply:
Reviewed - the Solicitor notifications have conditionals for disputed/undisputed.  If the variable was null it would cause an error when rendering the email.  I have removed the comments.